corehq/apps/hqwebapp/crispy.py

```diff
@@ -101,11 +85,10 @@ def render(self, form, context, template_pack=None, **kwargs):
                 template_pack=template_pack,
             )
         fields_html = mark_safe(fields_html)  # nosec: just concatenated safe fields
-        offsets = _get_offsets(context)
         context.update({
             'formactions': self,
             'fields_output': fields_html,
-            'offsets': offsets,
+            'offsets': '',
```

Comment:
Mabye, I'll take another look. I'm not sure how many places use (require?) it. It's hard to search for because it's a common word. I also need to check if crispy-boostrap3to5 needs it.

Reply:
https://github.com/dimagi/commcare-hq/pull/35519/commits/8549a39fa58c6a27bb9b6ce0e973e63b19c89efa